sdk/ai/azure-ai-generative/tests/synthetic_qa/e2etests/test_qa_data_generator_e2e.py

```diff
@@ -0,0 +1,179 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+import asyncio
+import logging
+
+import pytest
+from azure.ai.generative.synthetic.qa import QADataGenerator, QAType
+
+from utils import find_closest_prediction, f1_score
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.mark.e2etest
```

Comment:
```suggestion
@pytest.mark.e2etest
@pytest.mark.usefixtures("recorded_test")
```

This needs to request the "recorded_test" fixture so that it generates test recordings.